### PR TITLE
feat-added badges

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added Avatar and Icon Badges 
Also given different positions'  to Badges like top-left , bottom-right etc .
Live link :

### CODE REVIEW

File: componets/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@import url(../avatar/avatar.css);
+
+.badge__container {
+  position: relative;
+  margin: 1rem;
+}
+.badge {
+  position: absolute;
+  width: 0.93rem;
```
Comment (shivampandey0) [inline componets/badge/badge.css]: try following 8 point design system. (8 point grid )
don't be too specific with decimal values.

File: componets/badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
...tom-right%2520online%2522%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge__container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2520s%2522%2520src%253D%2522..%252F..%252Fassets%252F4.jpg%2522%2520href%253D%2522avatar1%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge%2520bottom-right%2520offline%2522%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge__container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2520s%2522%2520src%253D%2522..%252F..%252Fassets%252F4.jpg%2522%2520href%253D%2522avatar1%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge%2520bottom-right%2520dnd%2522%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
```
Comment (shivampandey0) [inline componets/badge/badge.html:82]: maybe shorten this URL

File: componets/badge/badge.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
```
Comment (shivampandey0) [inline componets/badge/badge.html]: font-awesome can be imported in CSS to be part of the component library and not to be needed to be explicitly linked in HTML

**Review by shivampandey0 — COMMENTED** [review]
Added some suggestions

File: componets/badge/badge.css (hunk 1 above)
Comment (prathmeshjagtap) [reply componets/badge/badge.css]: yes i did the change

File: componets/badge/badge.html (hunk 3 above)
Comment (prathmeshjagtap) [reply componets/badge/badge.html]: yes thanks for this i did the changes

File: componets/badge/badge.html (hunk 2 above)
Comment (prathmeshjagtap) [reply componets/badge/badge.html:82]: I tried to shorten it but the I frame is getting disturbed like its not working how I want it to work

**Comment by SanjayJatti** [comment]
Looking nice and clean. Go ahead and merge it.

**Pull request merged**